README.md

```diff
@@ -1,46 +1,87 @@
-# MODX Revolution
-
-[![Build Status](https://travis-ci.org/modxcms/revolution.svg?branch=2.x)](https://travis-ci.org/modxcms/revolution) [![Slack Chat](https://img.shields.io/badge/chat_in_slack-online-green.svg?longCache=true&style=flat&logo=slack)](https://modx.org)
+<p align="center">
+  <a href="http://modx.com/">
```

Comment:
```suggestion
  <a href="https//modx.com/">
```

Reply:
https://

Reply:
Missing a `:` there @JoshuaLuckers

Reply:
@JoshuaLuckers Maybe it makes sense for all links to write without a certificate?
For example:
<a href="//modx.com/">